src/node_modules/app/elasticsearch/types/genericTypes.js

```diff
@@ -76,13 +100,19 @@ export const is_object = is_shape(objectKey);
 const xorKey = 'xor';
 export const xorD = _shape(xorKey);
 export const xor = xorD();
-export const xorOf = mergeObj(xor);
+// export const xorOf = mergeObj(xor);
+export const xorOf = obj => mergeObj(xor)({
+	__selection: enumsForKeysOf(obj),
+	...obj
+});
```

Comment:
We'll probably need to remove `__selection` in a new pass, but for now I'd change to:

```suggestion
export const xorOf = obj => ({
	__selection: enumsForKeysOf(obj),
	...obj,
	...xor
});
```